content/en/user-guide/aws/cloudformation/index.md

```diff
@@ -104,211 +127,211 @@ Your feature requests help us prioritize which resources we need to prioritize,
-|AWS::RDS::DBInstance|✅|✅|-|
-|AWS::RDS::DBParameterGroup|✅|✅|-|
-|AWS::RDS::DBSubnetGroup|✅|✅|-|
-|AWS::Redshift::ClusterParameterGroup|✅|✅|-|
-|AWS::Redshift::ClusterSecurityGroup|✅|✅|-|
-|AWS::Redshift::ClusterSubnetGroup|✅|✅|-|
-|AWS::Route53::HostedZone|✅|✅|-|
-|AWS::SES::ReceiptRule|✅|✅|-|
-|AWS::SES::ReceiptRuleSet|✅|✅|-|
-|AWS::SES::Template|✅|✅|✅|
-|AWS::ServiceDiscovery::HttpNamespace|✅|✅|-|
-|AWS::ServiceDiscovery::PrivateDnsNamespace|✅|✅|-|
-|AWS::ServiceDiscovery::PublicDnsNamespace|✅|✅|-|
-|AWS::ServiceDiscovery::Service|✅|✅|-|
-|AWS::Timestream::Database|✅|✅|-|
-|AWS::Timestream::Table|✅|✅|-|
+| Intrinsic Function | Supported | Explanation                                                  |
+| ------------------ | --------- | ------------------------------------------------------------ |
+| `Fn::And`          | Yes       | Performs a logical AND operation on two or more expressions. |
+| `Fn::Or`           | Yes       | Performs a logical OR operation on two or more expressions.  |
+| `Fn::Base64`       | Yes       | Converts a binary string to a Base64-encoded string.         |
+| `Fn::Sub`          | Yes       | Performs a string substitution operation.                    |
+| `Fn::Split`        | Yes       | Splits a string into an array of strings.                    |
+| `Fn::Length`       | Yes       | Returns the length of a string.                              |
+| `Fn::Join`         | Yes       | Joins an array of strings into a single string.              |
+| `Fn::FindInMap`    | Yes       | Finds a value in a map.                                      |
+| `Fn::Ref`          | Yes       | References a resource in the template.                       |
+| `Fn::GetAtt`       | Yes       | Gets an attribute from a resource.                           |
+| `Fn::If`           | Yes       | Performs a conditional evaluation.                           |
+| `Fn::Import`       | Yes       | Imports a value from another template.                       |
+| `Fn::ToJsonString` | No        | Converts an object or map into a json string.                |
+| `Fn::Cidr`         | No        | Generates a CIDR block from the inputs.                      |
+| `Fn::GetAZs`       | No        | Returns a list of the Availability Zones of a region.        |
+
+### Resources
+
+{{< alert title="Note">}}
+When utilizing the Community Edition, any resources within the stack that are not supported will be disregarded and won't be deployed.
+{{< /alert >}}
+
+#### Community Edition
+
+| Resource                                    | Create | Delete | Update |
+| ------------------------------------------- | -----: | -----: | -----: |
+| AWS::ApiGateway::Account                    |      - |      - |      - |
+| AWS::ApiGateway::ApiKey                     |     ✅ |      - |      - |
+| AWS::ApiGateway::BasePathMapping            |     ✅ |      - |      - |
+| AWS::ApiGateway::Deployment                 |     ✅ |      - |      - |
```

Comment:
We should autogenerate this table - @dominikschubert that is/was something you were working on, right?